README.md

```diff
@@ -61,3 +61,21 @@ Convert a JSON representation back into an immutable object
 ### `transit.withFilter(function) => transit`
 
 Create a modified version of the transit API that deeply applies the provided filter function to all immutable collections before serialising. Can be used to exclude entries.
+
+### `transit.withRecords([recordClasses])[.withFilter(function)] => transit`
```

Comment:
No worries, I might take a crack at this myself - I think there should be a way to achieve this and end up with less code 😄